src/shared/interfaces/json-editor-config.ts

```diff
@@ -46,6 +46,13 @@ export interface JsonEditorConfig {
    */
   schemaOptions?: SchemaOptions;
 
+  /**
+   * If set, it hides the preview section of the editor.
+   *
+   * - It is `false` by default.
+   */
+  isPreviewerHidden?: boolean;
```

Comment:
can you rename it to `isPreviewerHiddenOnStart` ?

Reply:
I kept the property name that is used from the ui when shows/hides the previewer. There is also another option `isPreviewDisabled` so I tried not to create yet another one. But I agree is more self explained.

Reply:
Yeah, since it's not always hidden and can be toggled later, I'd go with explicit name.